build.gradle

```diff
@@ -406,6 +407,19 @@ allprojects {
         i++;
         folder.output = "build-eclipse/" + i
       }
+      /*
+       * Allow accessing com/sun/net/httpserver in projects that have
+       * configured forbidden apis to allow it.
+       */
+      plugins.withType(ForbiddenApisPlugin) {
```

Comment:
This probably shouldn't go inside `plugins.withType(JavaBasePlugin) {`. Nesting callbacks like this can have weird ordering consequences.